data/abilities.js

```diff
@@ -436,6 +436,7 @@ exports.BattleAbilities = {
 		},
 		onAnyDamage: function (damage, target, source, effect) {
 			if (effect && effect.id === 'aftermath') {
+				this.add('-activate', this.effectData.target, 'ability: Damp');
```

Comment:
Actually, it shouldn't be anything since neither Aftermath nor Damp activate when both are on the field and the Aftermath Pokemon is knocked out with a contact move. Aftermath just silently fails to work.